### ISSUE TITLE
Add default config for Bodyguard.Plug.Authorize plug options

### ISSUE THREAD

**Issue by schrockwell** [schrockwell/bodyguard#48]
Example:
```elixir
config :bodyguard, Bodyguard.Plug.Authorize,
  defaults: [
    action: {Phoenix.Controller, :current_action},
    user: {App, :get_current_user}
  ]
```

Note we probably shouldn't/can't use anonymous functions in config files.

These would be merged in with the plug options, which would take precedence.

**Comment by jung-hunsoo** [comment]
In Coherence, the function is Coherence.current_user(conn).
And I'm worry about the default action won't useful when the Bodyguard is in the context.
Buy the way defining defaults will be useful in productivity.

**Comment by schrockwell** [comment]
This is available in 2.4.0 now!

@jung-hunsoo yes, the default value for any option can be configured
